helpers/blog_helper.rb

```diff
@@ -3,15 +3,10 @@ module BlogHelper
   # Try to switch over to super.url_slug, if possible!
   def self.url_slug(post)
     # This is Split out for Testing Purposes.
-    return url_slug_from_wordpress_url(post&.wordpress_url) if post&.wordpress_url&.present?
+    return post.url_slug if post&.url_slug&.present?
```

Comment:
In the past, `url_slug` vs. `urlSlug` has been very confusing—you sure it's `url_slug`?  They changed their API to be camelCase on new fields, unless you overwrote it?  Anyways, I'm sure you've given it a go at some level.

Reply:
😬 url_slug seems to work locally

Reply:
You'll figure it out!